content/learn/book/getting-started/setup/_index.md

```diff
@@ -200,6 +200,9 @@ Bevy can be built just fine using default configuration on stable Rust. However
 
     For more information, see [The rustup book: Overrides](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file).
 
+* **Cranelift Codegen**: This uses a new nightly-only codegen that can be 300% faster than LLVM! However it only currently works on Linux and is generally still immature.
```

Comment:
30% is a more realistic average.

It works fine on Windows now and x86_64 macOS (arm64 macOS has some ABI issues)

Reply:
I could not get Bevy compiling on macOS with cranelift enabled. Specifically, `objc2` did not build.
Windows works, but the installation process is a bit whackier and it *may* not play nice with sccache.
After my little investigation, it seems like the only platform where everything runs smoothly out of the box is still Linux.

Reply:
An issue for that at https://github.com/rust-lang/rustc_codegen_cranelift/issues would be appreciated.

Reply:
Sure! I'll post one tomorrow :)

Reply:
Update: there you go https://github.com/rust-lang/rustc_codegen_cranelift/issues/1504
Let me know in that issue if there's anything I can try out to help debug it.

Reply:
Thanks!